lib/ui/text/text_box.h

```diff
@@ -1,28 +0,0 @@
-// Copyright 2013 The Flutter Authors. All rights reserved.
```

Comment:
Out of curiosity, how did you determine that this was dead code?

Reply:
It came up in the clang_tidy audit, and I grepped for `text_box.h` and didn't find any usages outside of metadata.

It looks like it was a Fuchsia-era thing looking at blame history?